gschem/src/x_newtext.c

```diff
@@ -75,7 +75,8 @@ dialog_response_apply (NewText *dialog)
   int align = LOWER_LEFT;
   int color = TEXT_COLOR;
   int rotate = 0;
-  int size = 12;
+  GschemToplevel *w_current = GSCHEM_DIALOG (dialog)->w_current;
```

Comment:
Speaking of refactoring, would it be possible to [do it](https://github.com/graahnul-grom/lepton-eda/commit/6367613187244a4bed046541a243775bcc3e16a8) for `x_window_create_main()`?
The code there in its current state makes it almost impossible to experiment with
main window layout and widgets.